src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,77 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private int buySum = 0;
+    private int supplySum = 0;
+    private final String supplyOperation = "supply";
+    private final String buyOperation = "buy";
+    private final String result = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] tempString = readFile(fromFileName).toString().split("\\W+");
+        calculateSum(tempString);
+        checkWriteFile(toFileName);
```

Comment:
what's the purpose of it? you can just writeToFile without checks

Reply:
I created this check for already existing files...I thought it's a good idea... but then I have changed writeFile method and this check became useless... I left this method because I want to discuss with the mentor this solution for checking if file already exists and isn't empty...